gips/0082-gip-supported-networks-registry.md

```diff
@@ -0,0 +1,107 @@
+---
+GIP: 0082
```

Comment:
GIP number needs to be quoted to avoid being treated as a number when parsing (resulting in 0s dropped).
```suggestion
GIP: "0082"
```